content/projects/gsoc/2023/project-ideas/docker-compose-build.adoc

```diff
@@ -1,10 +1,10 @@
 ---
 layout: gsocprojectidea
-title: "Building Jenkins.io with docker-compose"
+title: "Jenkins docker examples to help users get started"
 goal: "(to be clarified)"
 category: Tools
 year: 2023
-status: discussion
+status: draft
```

Comment:
```suggestion
status: discussion
```
(To be restored)

Reply:
The proposal can be kept as `draft`. The change on the change is not useful.